RationaleMCP/ReadMe.md

```diff
@@ -50,7 +50,7 @@ but the rest of the development on a branch/pull-request before being accepted.
 |Active|0012|Calling blocks as functions|[MCP/0012](https://github.com/modelica/ModelicaSpecification/tree/MCP/0012/RationaleMCP/0012)||
 |On hold|0011|Allow user-defined functions in reductions|||
 |On hold|0010|Adding guards to reductions|||
-|Active|0009|Undefined modification|[MCP/0009](https://github.com/modelica/ModelicaSpecification/tree/MCP/0009/RationaleMCP/0009)|https://github.com/modelica/ModelicaSpecification/pull/3167|
+|Added in [3.6](https://github.com/modelica/ModelicaSpecification/releases/tag/v3.6)|0009|Undefined modification|[MCP/0009](https://github.com/modelica/ModelicaSpecification/tree/MCP/0009/RationaleMCP/0009)|https://github.com/modelica/ModelicaSpecification/pull/3167|
```

Comment:
Save as above.

Reply:
Resolving according to yesterday's phone meeting decision to use _master_ as the release preparation branch.